src/qml/geometry_editors/VertexEditorToolbar.qml

```diff
@@ -27,11 +27,18 @@ VisibilityFadingRow {
     featureModel.vertexModel.reset()
   }
 
+  function applyChanges( apply ) {
+    if ( apply && featureModel.vertexModel.dirty ){
+      featureModel.applyVertexModelToGeometry()
+      featureModel.save()
```

Comment:
OK so here, you'll have to re-create the vertex model based on modified feature geometry. It's needed due to the way we (now) keep track of new vs modified vertices in the context of topological editing.

Reply:
Okay, I'll check. Thanks.